crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
```

Comment:
If you really need a two-way lookup then I'd suggest probably a list of tuples is better than a dict. Although as below I'm not sure why we need to look up a code based on a full coin name.